test/mavsdk_tests/process_helper.py

```diff
@@ -190,7 +190,41 @@ def __init__(self,
         self.cmd = "gzserver"
         self.args = ["--verbose",
                      workspace_dir + "/Tools/sitl_gazebo/worlds/" +
-                     self.model + ".world"]
+                     "empty.world"]
+
+    def add_to_env_if_set(self, var: str) -> None:
+        if var in os.environ:
+            self.env[var] = os.environ[var]
+
+
+class GzmodelspawnRunner(Runner):
+    def __init__(self,
+                 workspace_dir: str,
+                 log_dir: str,
+                 model: str,
+                 case: str,
+                 speed_factor: float,
+                 verbose: bool):
+        super().__init__(log_dir, model, case, verbose)
+        self.name = "gz"
+        self.cwd = workspace_dir
+        self.env = {"PATH": os.environ['PATH'],
+                    "HOME": os.environ['HOME'],
+                    "GAZEBO_PLUGIN_PATH":
+                    workspace_dir + "/build/px4_sitl_default/build_gazebo",
+                    "GAZEBO_MODEL_PATH":
+                    workspace_dir + "/Tools/sitl_gazebo/models",
+                    "PX4_SIM_SPEED_FACTOR": str(speed_factor),
```

Comment:
Is this one required?

Reply:
To keep the way the tester runs server model spawning together, I replaced it with a bash script!